index.html

```diff
@@ -230,7 +230,31 @@ <h5 class="font-weight-bold">Building Softwares</h5>
             
         </div>
     </section>
+<section>
+    <section>
+        <div>
+        <div class="card mb-3" style="max-width: 540px;">
+          <div class="row no-gutters">
+            <div class="col-md-4 ">
+              <img class="resume" src="resume.png" class="card-img" alt="Resume_image">
+            </div>
+            <div class="col-md-8">
+              <div class="card-body mod-card-body">
+                <h5 class="card-title heading ">Grab my Resume</h5>
+                <a class="btn btn-primary"
+                  href="resume.png" target="blank"
+                  role="button" download="resume">Download</a>
+                <a class="btn btn-primary"
+                  href="resume.png" target="blank"
```

Comment:
Always put the images inside the `assets/images/home` directory for better management of assets